CHANGELOG.md

```diff
@@ -17,8 +17,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ### Technical Changes
 
-## [4.4.0]
+- Dropped support for older Helm versions: For rendering the Helm charts stackrox-central-services and
```

Comment:
I wonder if this should:
- be in the "removed features" section?
- mention the `allowUnsupportedHelmVersion` workaround?
🤔

Reply:
I'm fine with moving this into the "Removed Features" section, but I would advise against mentioning this escape hatch here. Otherwise it would seem kinda weird: "Here, our Helm charts don't work anymore with Helm x.y.z. But if you set this flag, then they kinda do" :-)

I *could* imagine situations in which something like this comes to the rescue of a user, hence I put it in.